docs/run.md

```diff
@@ -128,11 +128,13 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete. Gallery images can be found according to the instructions [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage)
```

Comment:
This very same URI is already in the repo. Not against putting in again, but... why no use the right/fixed term "Marketplace"?

Reply:
Sorry, I was going off the variable name. This looks a little confusing now though since the variable is "gallery_image"

Reply:
It is not